stencil.config.ts

```diff
@@ -24,6 +25,11 @@ export const config: Config = {
     slotChildNodesFix: true,
   },
   outputTargets: [
+    reactOutputTarget({
+      componentCorePackage: '../dist/types',
+      proxiesFile: './react-bindings/components.ts',
+      includeDefineCustomElements: false,
```

Comment:
I'm going against the default here because:

1. We are already [calling `defineCustomElements` in `vets-website`](https://github.com/department-of-veterans-affairs/vets-website/blob/294c69152bdf6728207091e18564b83b7dae7184/src/platform/site-wide/wc-loader.js#L5-L7) so that the plain Web Components will work.
1. Related to my earlier comment on the `componentCorePackage` - if this is true, then we try to import from `${componentCorePackage}/loader`, and `dist` and `loader` are siblings rather than parent/child